src/main/java/it/pagopa/pn/commons/log/PnAuditLogEvent.java

```diff
@@ -0,0 +1,19 @@
+package it.pagopa.pn.commons.log;
+
+public class PnAuditLogEvent {
+    PnAuditLogEventType type;
+    String message;
+    String format;
+    Object[] arguments;
+
+    public PnAuditLogEvent(PnAuditLogEventType type, String message) {
+        this.type = type;
+        this.message = message;
+    }
+
+    public PnAuditLogEvent(PnAuditLogEventType type, String format, Object... arguments) {
+        this.type = type;
+        this.format = format;
+        this.arguments = arguments;
+    }
+}
```

Comment:
Mio sembra che manchi la parte di definizione degli eventi di successo o fallimento.
Come mai hai scelto di non metterla?

Reply:
Ho scelto di metterla nel metodo di log perché più pratico: invece di modificare l'evento e chiamare il metodo è più semplice chiamare il metodo con un parametro in più.